.circleci/config.yml

```diff
@@ -50,12 +77,19 @@ jobs:
       - run: echo "!TODO"
 
 workflows:
-  build_and_test:
+  Recipe_app_workflow:
```

Comment:
hmm, not sure about the name change here. I'd leave the previous one I guess. If you plan to have 1 workflow only then I guess this name is ok, but personally I would go for a separate workflow for the deployment